src/main/java/serialization/WriteObject.java

```diff
@@ -9,8 +9,8 @@ public class WriteObject {
     private final static String PEOPLE_DATA = "people.bin";
```

Comment:
Reorder the modifiers to comply with the Java Language Specification.

Correct order: `static final`